### DISCUSSION TITLE
Install OpenDC locally - BUILD FAILED

### DISCUSSION THREAD

**Discussion by mochamadrizal26** [atlarge-research/opendc#146] [General]
Hi, I have a question. I am trying to install locally the OpenDC.
But I have this problem.

Execution failed for task ':buildSrc:generateExternalPluginSpecBuilders'.
> Could not resolve all files for configuration ':buildSrc:compileClasspath'.
   > Could not resolve io.quarkus:gradle-application-plugin:2.16.5.Final.
     Required by:
         project :buildSrc
      > No matching variant of io.quarkus:gradle-application-plugin:2.16.5.Final was found. The consumer was configured to find a library for use during compile-time, compatible with Java 8, preferably not packaged as a jar, preferably optimized for standard JVMs, and its dependencies declared externally, as well as attribute 'org.gradle.plugin.api-version' with value '8.0', attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm' but:

I followed the guideline and use Java 17.

Are there any prerequisites that I miss to install?
Your suggestion and help would be appreciated.

**Comment by fabianishere** [comment]
It seems Gradle is still building with Java 8. How are you running the build using Java 17?

**Reply by mochamadrizal26** [reply]
After I checked again, my gradle default in IntelliJ pointed to Java 8.
After changing it to Java 17. The build was successful.
Thanks.

![image](https://user-images.githubusercontent.com/39995309/235652173-32604ad9-4330-4598-b3a3-8ee52157e3b7.png)
